### DISCUSSION TITLE
iOS Feedback/App Recommendation

### DISCUSSION THREAD

**Discussion by Woshimbo** [community/community#47002] [Other Feature Feedback, Questions, & Ideas]
Awesome job so far, I love being able to work from the app and integrations makes it so much easier.

I'd just like to please suggest having the file size and number of files added to the "browse code" section. "Working Copy" would be a good example.

I'd also like to mention "Editorial" and "Pythonista". If you're stuck doing this kind of stuff with only a phone, they're life savers. Editorial also works very nicely with the GitHub app.

It would also be very nice to be able to add and delete files, etc from within the GitHub app as well as have access to the wiki.

**Comment by Donzke45** [comment]
alt+click/return

**Comment by Abdul700160** [comment]
Hello,
Thanks for your efforts.
On iOS Keka App I didn't find or miss the option to compress my photos directly by selecting the needed ones and then select Share option and then an option to Compress those files by Keka. 

Please advise.
best regards.
